cmd/proxy/actions/app.go

```diff
@@ -16,8 +16,10 @@ import (
 	"github.com/gocraft/work"
 	"github.com/gomods/athens/pkg/config/env"
 	"github.com/gomods/athens/pkg/log"
+	gomodsmiddleware "github.com/gomods/athens/pkg/middleware"
```

Comment:
`pkgmiddleware` or just `mw` is probably cleaner.

Reply:
Will change!